tests/testsuite/build_auth.rs

```diff
@@ -105,7 +105,7 @@ fn http_auth_offered() {
             authors = []
 
             [dependencies.bar]
-            git = "http://127.0.0.1:{}/foo/bar"
+            git = "https://127.0.0.1:{}/foo/bar"
```

Comment:
This causes the error to be "SSL error: received early EOF; class=Net (12); code=Eof (-20)"

Reply:
Fixed in ee20f73